pkgs/development/libraries/opencv/3.x.nix

```diff
@@ -110,8 +94,12 @@ stdenv.mkDerivation rec {
     (opencvFlag "OPENEXR" enableEXR)
     (opencvFlag "CUDA" enableCuda)
     (opencvFlag "CUBLAS" enableCuda)
-  ] ++ lib.optionals enableContrib [ "-DOPENCV_EXTRA_MODULES_PATH=${contribSrc}/modules" ]
-    ++ lib.optionals enableCuda [ "-DCUDA_FAST_MATH=ON" ];
+  ] ++ lib.optionals enableContrib [
+    "-DOPENCV_EXTRA_MODULES_PATH=${contribSrc}/modules"
+    # the following modules are badly behaved and attempt to download 3rd party libs as part of the build process:
+    "-DBUILD_opencv_dnn=OFF"          # attempts to download protobuf src and googlenet caffe model
```

Comment:
Can we change the Makefile script using `sed` to remove the command to download neural nets?